interactions/api/events/discord.py

```diff
@@ -751,3 +756,56 @@ class GuildAuditLogEntryCreate(GuildEvent):
 
     audit_log_entry: interactions.models.AuditLogEntry = attrs.field(repr=False)
     """The audit log entry object"""
+
+
+@attrs.define(eq=False, order=False, hash=False, kw_only=False)
+class GuildScheduledEventCreate(BaseEvent):
+    """Dispatched when scheduled event is created"""
+
+    scheduled_event: interactions.models.ScheduledEvent = attrs.field(repr=True)
```

Comment:
We're using the `Absent["type"]`/`Optional["type"]`/`"type"` convention for string-type parsing everywhere else in our code. Sequential proposals suggest moving over to it

```suggestion
    scheduled_event: "ScheduledEvent" = attrs.field(repr=True)
```